### ISSUE TITLE
Store bin size in dict file

### ISSUE THREAD

**Issue by justinjjvanderhooft** [sdrogers/ms2ldaviz#143]
Store the bin size in dict file for uploading purposes (it does not make sense to use a different binning size as the one with which the experiment was made)

**Comment by joewandy** [comment]
Is this to store the bin size in GNPS-created experiments?

**Comment by justinjjvanderhooft** [comment]
Yes, if we update that in the GNPS-MS2LDA workflow, then the user doesn't need to indicate that upon uploading into the MS2LDA web app....
